
---
sensor_msgs/CompressedImage[] cropped_images
